packages/expo-firebase-recaptcha/package.json

```diff
@@ -40,5 +40,10 @@
   },
   "peerDependencies": {
     "react-native-webview": "*"
+  },
```

Comment:
@IjzerenHein - i added back peerDependencies here and on expo-auth-session and expo-standard-web-crypto, but used the optional flag on peerDependenciesMeta as suggested by @byCedric so npm 7 will not attempt to auto install them. [more info](https://github.com/npm/rfcs/issues/221)

will this be sufficient for our requirements on snack?

Reply:
Yep I think so 👍